qc_grader/challenges/fall_2022/lab3.py

```diff
@@ -1,33 +1,33 @@
 from typeguard import typechecked
+
+from typing import Callable
+
 from qiskit import QuantumCircuit
+from qiskit.result import QuasiDistribution
 from qc_grader.grader.grade import grade, get_problem_set
+import random
 
 _challenge_id = 'fall_2022'
 
 @typechecked
-def grade_lab3_ex1(qc: QuantumCircuit) -> None:
-    grade(qc, 'ex3-1', _challenge_id)
+def grade_lab3_ex1(attempt_qc: QuantumCircuit, attempt_n: int) -> None:
+    answer = {
+        'attempt_qc': attempt_qc,
+        'attempt_n': attempt_n
+    }
+    grade(answer, 'ex3-1', _challenge_id)
 
 @typechecked
-def grade_lab3_ex2(qc: QuantumCircuit) -> None:
-    grade(qc, 'ex3-2', _challenge_id)
+def grade_lab3_ex2(attempt_ising: PauliSumOp) -> None:
+    grade(attempt_ising, 'ex3-2', _challenge_id)
 
-@typechecked
-def grade_lab3_ex3(qc: QuantumCircuit) -> None:
-    grade(qc, 'ex3-3', _challenge_id)
-
-@typechecked
-def grade_lab3_ex4(qc: QuantumCircuit) -> None:
-    grade(qc, 'ex3-4', _challenge_id)
-
-@typechecked
-def grade_lab3_ex5(qc: QuantumCircuit) -> None:
-    grade(qc, 'ex3-5', _challenge_id)
-
-@typechecked
-def grade_lab3_ex6(qc: QuantumCircuit) -> None:
-    grade(qc, 'ex3-6', _challenge_id)
 
 @typechecked
-def grade_lab3_ex7(qc: QuantumCircuit) -> None:
-    grade(qc, 'ex3-7', _challenge_id)
```

Comment:
`fall_2022/__init__.py` needsto be updated accordingly to coincide with updates here